sotodlib/toast/workflows/proc_mapmaker.py

```diff
@@ -59,10 +59,21 @@ def setup_mapmaker(operators, templates):
             fit_subharmonics=False,
         )
     )
-    operators.append(toast.ops.BinMap(name="binner", pixel_dist="pix_dist"))
     operators.append(
         toast.ops.BinMap(
-            name="binner_final", enabled=False, pixel_dist="pix_dist_final"
+            name="binner",
+            pixel_dist="pix_dist",
+            # Discard invalid data and turnarounds by default
+            shared_flag_mask=3,
```

Comment:
The default shared flag mask for the BinMap operator is to [cut all "non-science" data](https://github.com/hpc4cmb/toast/blob/697eefd199d61c25c082535078875bd3973256bf/src/toast/ops/mapmaker_binning.py#L83).  The hard-coded value here is equivalent to including the `processing` bit (which I agree with) and excluding the `irregular` and `unstable_scanrate` bits (why?).  It looks like we should actually change the meaning of the non-science flag in toast to include processing bit (which it does for the detector mask).

If you need a temporary workaround here in sotodlib, could we do something like
```
shared_flag_mask=(defaults.shared_mask_nonscience | defaults.shared_mask_processing)
```
to make it more clear the names of the flag bits?